src/api/posts.ts

```diff
@@ -0,0 +1,25 @@
+import { Post } from '../react-app-env';
+
+export const BASE_URL = 'https://mate.academy/students-api/';
+
+export async function getPosts(userId: string): Promise<Post[]> {
+  const response = await fetch(`${BASE_URL}/posts?userId=${userId}`);
+
+  if (!response.ok) {
+    throw new Error(`${response.status} - ${response.statusText}`);
+  }
+
+  return response.json();
+}
+
+export async function getPostbyId(postId:number) {
```

Comment:
Provide an explicit return type in a function declaration.
Use camelCase when naming objects, functions, and instances. Check the [doc](https://mate-academy.github.io/style-guides/javascript.html#232-use-camelcase-when-naming-objects-functions-and-instances-function-names-are-typically-verbs-or-verb-phrases).
Add a space between ```:``` and ```number``` to follow the [style guide](https://mate-academy.github.io/style-guides/javascript.html).
```suggestion
export async function getPostById(postId: number): Promise<Post> {
```